.cspell.json

```
@@ -3,6 +3,7 @@
   "language": "en",
   "allowCompoundWords": false,
   "ignorePaths": [
+    "**.*.dxf",
```

Is this correct? :thinking: 

```suggestion
    "**/*.dxf",
```

---

Sorry, that's typo.